### ISSUE TITLE
TRTEXEC covert failure issue with ERROR: ModelImporter.cpp:296 In function importModel:

### ISSUE THREAD

**Issue by quxinqq** [NVIDIA/TensorRT#1431]
## Description

When I use trxexec to convert my onnx model which was coverted by keras2onnx plugin. I encountered below issue, can you please help to guide. Thanks.
WARNING: ONNX model has a newer ir_version (0.0.7) than this parser was built against (0.0.3).
While parsing node number 1 [BatchNormalization]:
ERROR: ModelImporter.cpp:296 In function importModel:
[5] Assertion failed: tensors.count(input_name)
[07/04/2021-10:25:20] [E] Failed to parse onnx file
[07/04/2021-10:25:20] [E] Parsing model failed
[07/04/2021-10:25:20] [E] Engine could not be created
&&&& FAILED TensorRT.trtexec # /home/xaqu/software_package/TensorRT-6.0.1.5/bin/trtexec --onnx=/home/xaqu/notebook_dir/SLA/model_best.onnx --saveEngine=/home/xaqu/notebook_dir/SLA/resnet_engine.trt --explicitBatch

## Environment

**TensorRT Version**: TensorRT-6.0.1.5
**NVIDIA GPU**: Tesla V100-PCIE
**NVIDIA Driver Version**: 418.211.00
**CUDA Version**: 10.1
**CUDNN Version**: 7.6.5
**Operating System**: 3.10.0-957.27.2.el7.x86_64 (mockbuild@kbuilder.bsys.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-36) 
**Python Version (if applicable)**: 3.7.6 with anaconda
**Tensorflow Version (if applicable)**: 2.2.0
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Relevant Files

[keras model and transfered onnx model.zip](https://github.com/NVIDIA/TensorRT/files/6928180/keras.model.and.transfered.onnx.model.zip)

## Steps To Reproduce
Step1. keras model summary: model.summary()

![图片](https://user-images.githubusercontent.com/88089887/128113305-0451fcd2-c1ad-4fc6-9fa7-0e5460e5b54e.png)

Step2 keras h5 foramt model coverted to onnx model by keras2onnx
import os
import numpy as np
import tensorflow as tf
from tensorflow.python.compiler.tensorrt import trt_convert as trt
import onnx
model=load_model('/home/xaqu/notebook_dir/SLA/saved_models/training_mcs_20210618-153954/model_best.h5')
onnx_model = keras2onnx.convert_keras(model, model.name)
onnx.save_model(onnx_model, '/home/xaqu/notebook_dir/SLA/model_best.onnx')
![图片](https://user-images.githubusercontent.com/88089887/128112193-de3eedba-ce5b-4a00-8a57-566ccc64ffaa.png)

Step 2: Use trtexec to covert onnx model to a TensorRT engine
/home/xaqu/software_package/TensorRT-6.0.1.5/bin/trtexec --onnx=/home/xaqu/notebook_dir/SLA/model_best.onnx --saveEngine=/home/xaqu/notebook_dir/SLA/resnet_engine.trt  --explicitBatch

[07/04/2021-10:25:20] [I] === Model Options ===
[07/04/2021-10:25:20] [I] Format: ONNX
[07/04/2021-10:25:20] [I] Model: /home/xaqu/notebook_dir/SLA/model_best.onnx
[07/04/2021-10:25:20] [I] Output:
[07/04/2021-10:25:20] [I] === Build Options ===
[07/04/2021-10:25:20] [I] Max batch: explicit
[07/04/2021-10:25:20] [I] Workspace: 16 MB
[07/04/2021-10:25:20] [I] minTiming: 1
[07/04/2021-10:25:20] [I] avgTiming: 8
[07/04/2021-10:25:20] [I] Precision: FP32
[07/04/2021-10:25:20] [I] Calibration: 
[07/04/2021-10:25:20] [I] Safe mode: Disabled
[07/04/2021-10:25:20] [I] Save engine: /home/xaqu/notebook_dir/SLA/resnet_engine.trt
[07/04/2021-10:25:20] [I] Load engine: 
[07/04/2021-10:25:20] [I] Inputs format: fp32:CHW
[07/04/2021-10:25:20] [I] Outputs format: fp32:CHW
[07/04/2021-10:25:20] [I] Input build shapes: model
[07/04/2021-10:25:20] [I] === System Options ===
[07/04/2021-10:25:20] [I] Device: 0
[07/04/2021-10:25:20] [I] DLACore: 
[07/04/2021-10:25:20] [I] Plugins:
[07/04/2021-10:25:20] [I] === Inference Options ===
[07/04/2021-10:25:20] [I] Batch: Explicit
[07/04/2021-10:25:20] [I] Iterations: 10 (200 ms warm up)
[07/04/2021-10:25:20] [I] Duration: 10s
[07/04/2021-10:25:20] [I] Sleep time: 0ms
[07/04/2021-10:25:20] [I] Streams: 1
[07/04/2021-10:25:20] [I] Spin-wait: Disabled
[07/04/2021-10:25:20] [I] Multithreading: Enabled
[07/04/2021-10:25:20] [I] CUDA Graph: Disabled
[07/04/2021-10:25:20] [I] Skip inference: Disabled
[07/04/2021-10:25:20] [I] === Reporting Options ===
[07/04/2021-10:25:20] [I] Verbose: Disabled
[07/04/2021-10:25:20] [I] Averages: 10 inferences
[07/04/2021-10:25:20] [I] Percentile: 99
[07/04/2021-10:25:20] [I] Dump output: Disabled
[07/04/2021-10:25:20] [I] Profile: Disabled
[07/04/2021-10:25:20] [I] Export timing to JSON file: 
[07/04/2021-10:25:20] [I] Export profile to JSON file: 
[07/04/2021-10:25:20] [I] 
----------------------------------------------------------------
Input filename:   /home/xaqu/notebook_dir/SLA/model_best.onnx
ONNX IR version:  0.0.7
Opset version:    13
Producer name:    keras2onnx
Producer version: 1.8.1
Domain:           onnxmltools
Model version:    0
Doc string:       
----------------------------------------------------------------
WARNING: ONNX model has a newer ir_version (0.0.7) than this parser was built against (0.0.3).
While parsing node number 1 [BatchNormalization]:
ERROR: ModelImporter.cpp:296 In function importModel:
[5] Assertion failed: tensors.count(input_name)
[07/04/2021-10:25:20] [E] Failed to parse onnx file
[07/04/2021-10:25:20] [E] Parsing model failed
[07/04/2021-10:25:20] [E] Engine could not be created
&&&& FAILED TensorRT.trtexec # /home/xaqu/software_package/TensorRT-6.0.1.5/bin/trtexec --onnx=/home/xaqu/notebook_dir/SLA/model_best.onnx --saveEngine=/home/xaqu/notebook_dir/SLA/resnet_engine.trt --explicitBatch

**Comment by quxinqq** [comment]
When I downgrade onnx ir version to 0.0.0.3, the failure still happen.

[07/04/2021-11:01:06] [I] === Model Options ===
[07/04/2021-11:01:06] [I] Format: ONNX
[07/04/2021-11:01:06] [I] Model: /home/xaqu/notebook_dir/SLA/model_best_ir_version_3.onnx
[07/04/2021-11:01:06] [I] Output:
[07/04/2021-11:01:06] [I] === Build Options ===
[07/04/2021-11:01:06] [I] Max batch: explicit
[07/04/2021-11:01:06] [I] Workspace: 16 MB
[07/04/2021-11:01:06] [I] minTiming: 1
[07/04/2021-11:01:06] [I] avgTiming: 8
[07/04/2021-11:01:06] [I] Precision: FP32
[07/04/2021-11:01:06] [I] Calibration: 
[07/04/2021-11:01:06] [I] Safe mode: Disabled
[07/04/2021-11:01:06] [I] Save engine: /home/xaqu/notebook_dir/SLA/resnet_engine.trt
[07/04/2021-11:01:06] [I] Load engine: 
[07/04/2021-11:01:06] [I] Inputs format: fp32:CHW
[07/04/2021-11:01:06] [I] Outputs format: fp32:CHW
[07/04/2021-11:01:06] [I] Input build shapes: model
[07/04/2021-11:01:06] [I] === System Options ===
[07/04/2021-11:01:06] [I] Device: 0
[07/04/2021-11:01:06] [I] DLACore: 
[07/04/2021-11:01:06] [I] Plugins:
[07/04/2021-11:01:06] [I] === Inference Options ===
[07/04/2021-11:01:06] [I] Batch: Explicit
[07/04/2021-11:01:06] [I] Iterations: 10 (200 ms warm up)
[07/04/2021-11:01:06] [I] Duration: 10s
[07/04/2021-11:01:06] [I] Sleep time: 0ms
[07/04/2021-11:01:06] [I] Streams: 1
[07/04/2021-11:01:06] [I] Spin-wait: Disabled
[07/04/2021-11:01:06] [I] Multithreading: Enabled
[07/04/2021-11:01:06] [I] CUDA Graph: Disabled
[07/04/2021-11:01:06] [I] Skip inference: Disabled
[07/04/2021-11:01:06] [I] === Reporting Options ===
[07/04/2021-11:01:06] [I] Verbose: Disabled
[07/04/2021-11:01:06] [I] Averages: 10 inferences
[07/04/2021-11:01:06] [I] Percentile: 99
[07/04/2021-11:01:06] [I] Dump output: Disabled
[07/04/2021-11:01:06] [I] Profile: Disabled
[07/04/2021-11:01:06] [I] Export timing to JSON file: 
[07/04/2021-11:01:06] [I] Export profile to JSON file: 
[07/04/2021-11:01:06] [I] 
----------------------------------------------------------------
Input filename:   /home/xaqu/notebook_dir/SLA/model_best_ir_version_3.onnx
ONNX IR version:  0.0.3
Opset version:    13
Producer name:    keras2onnx
Producer version: 1.8.1
Domain:           onnxmltools
Model version:    0
Doc string:       
----------------------------------------------------------------
While parsing node number 1 [BatchNormalization]:
ERROR: ModelImporter.cpp:296 In function importModel:
[5] Assertion failed: tensors.count(input_name)
[07/04/2021-11:01:06] [E] Failed to parse onnx file
[07/04/2021-11:01:06] [E] Parsing model failed
[07/04/2021-11:01:06] [E] Engine could not be created
&&&& FAILED TensorRT.trtexec # /home/xaqu/software_package/TensorRT-6.0.1.5/bin/trtexec --onnx=/home/xaqu/notebook_dir/SLA/model_best_ir_version_3.onnx --saveEngine=/home/xaqu/notebook_dir/SLA/resnet_engine.trt --explicitBatch

**Comment by galagam** [comment]
You're using a quite old version of TensorRT. Please attempt to reproduce with TensorRT 8.x.

**Comment by nvpohanh** [comment]
Closing for now due to >14 days with no response. Please feel free to reopen if the issue still exists. Thanks
